src/DaxkoSSOClient.php

```diff
@@ -108,6 +108,39 @@ public function getRequest($uri) {
 
 
   }
+  
+  /**
+   * Execute POST request to Daxko API.
+   *
+   * @param string $uri
+   *   Daxko API endpoint.
+   * @param array $body
+   *   An array of items for the request body.
+   *
+   * @return mixed
+   *   The response or error.
+   */
+  public function postRequest($uri, array $body = []) {
+
+    $token = $this->getDaxkoPartnerToken();
+
+    try {
```

Comment:
Hi, @froboy .
It is better do not copy that code and try to make it reusable.
For example, add method like `request` and use it in getRequest and postRequest

Reply:
@anpolimus I'd started that way and then reverted, so thanks for the push to do it right. https://github.com/ymcatwincities/daxko_sso/pull/3/commits/50b561e6ecda402515f6b22aa0ac7a4f9821c293